script/Arcueid/Day 6/06_00_ARC06_1_2.txt

```diff
@@ -302,27 +309,29 @@ She has a dignified posture, a clear tone of voice, and eyes that never waver. E
 [2380e70d4dfa6bebe9c8f70615956ff3ef0959d4]{
 -- Page 104, Offset 14177.
 -- がたん、とテーブルに足をひっかけて、秋葉は立ちあがった。
-Akiha stands up, hitting her foot on the table.
+// がたん、とテーブルに足をひっかけて、秋葉は立ちあがった。
+Akiha leaps to her feet, banging her shin against the table as she does so.
 }
 [555de0cc3511d410b8c5608493c787d4887146d8]{
 -- Page 105, Offset 14178.
 -- 「いや、太いってのは違うか。キリッとしてるから印象が強いのかな？　あんまり眉間に皺とかよせると、よくないぞ」
-"No, maybe thick isn't the right word. Maybe you leave a strong impression because they're tensed up. It's not good if you wrinkle your brows so much."
+"No, on second thought thick isn't the right word. Maybe they just give off that impression since they're all tensed up? It's not healthy to furrow your brow so much, you know."
 }
 [878a68dad00ae1f7501d8e17bacc6ce703e83027]{
 -- Page 105, Offset 14179.
 -- ただでさえ迫力ある美人なのに、視線まできつかったら並の男は逃げ出すからな？
-Even if she's a beautiful woman with a lot of power, I think most men would run away with a strong gaze like that.
+// A little context is lost around the まできつかったら perhaps? な？
+She may be a powerful and beautiful woman, but a glower like that would send a normal man running.
```

Comment:
maybe "scowl"? glower is a tad archaic

When I googled 'glower' for synonyms google did 'did you mean _flower_'